RGLServerPlugin/src/Scene.cc

```diff
@@ -130,6 +130,18 @@ bool RGLServerPluginManager::RemoveEntityFromRGLCb(
     entitiesInRgl.erase(entity);
     return true;
 }
+
+// always returns true, because the ecm will stop if it encounters false
```

Comment:
This comment may be confusing - I see that false is also returned below in specific case?